### PR TITLE
First draft pipeline with all steps 

### PR DESCRIPTION
**Pull request by praveenraj2018**
This is a draft pipeline with all steps implemented

### CODE REVIEW

**Comment by maxulysse** [comment]
Why is there one `rnavar.nf` script and one `rnavar_bk.nf` script?

**Comment by praveenraj2018** [comment]
> Why is there one `rnavar.nf` script and one `rnavar_bk.nf` script?

I will have to remove the *bk.nf. Taken as a backup.

**Comment by maxulysse** [comment]
OK, I think the last thing that we now need would be to fix the nf-core lint.

**Comment by praveenraj2018** [comment]
> OK, I think the last thing that we now need would be to fix the nf-core lint.

Yes, That's what I am doing now...

**Comment by maxulysse** [comment]
Don't hesitate to message me on Slack if you need help with that

File: .nf-core-lint.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+files_unchanged:
+  - .github/ISSUE_TEMPLATE/bug_report.md
+  - .github/ISSUE_TEMPLATE/feature_request.md
+  - .github/PULL_REQUEST_TEMPLATE.md
+  - .gitignore
+  - assets/nf-core-rnavar_logo.png
+  - docs/images/nf-core-rnavar_logo.png
```
Comment (maxulysse) [inline .nf-core-lint.yml:7]:
```suggestion
```
I don't think you need these files to be out of the nf-core linting

File: Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ COPY environment.yml /
 RUN conda env create --quiet -f /environment.yml && conda clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
-ENV PATH /opt/conda/envs/nf-core-rnavar-1.0dev/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-rnavar-3.1/bin:$PATH
 
 # Dump the details of the installed packages to a file for posterity
-RUN conda env export --name nf-core-rnavar-1.0dev > nf-core-rnavar-1.0dev.yml
+RUN conda env export --name nf-core-rnavar-3.1 > nf-core-rnavar-3.1.yml
```
Comment (maxulysse) [inline Dockerfile:13]: This file can be deleted

File: environment.yml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-  - conda-forge::pygments=2.5.2
+  - conda-forge::python=3.9.5
+  - conda-forge::markdown=3.3.4
+  - conda-forge::pymdown-extensions=8.2
+  - conda-forge::pygments=2.9.0
   # TODO nf-core: Add required software dependencies here
-  - bioconda::fastqc=0.11.8
-  - bioconda::multiqc=1.7
+  - bioconda::fastqc=0.11.9
+  - bioconda::multiqc=1.11
```
Comment (maxulysse) [inline environment.yml:15]: This file can be deleted too

**Review by maxulysse — CHANGES_REQUESTED** [review]
Looking very good.
Just need to remove the dockerfile and environment.yml file and I think we're good with this PR

**Pull request merged**